src/main/java/de/hs_mannheim/informatik/ct/persistence/services/RoomService.java

```diff
@@ -75,6 +76,17 @@ public Room saveRoom(@NonNull Room room) {
         return roomList;
     }
 
+    private void checkRoomPinFormatForRoom(Room room) {
```

Comment:
Should I remove the check i made earlier? i thought at first that the roomPin can be changed by a user, if it is not accessible by him, i think it isn't usefull.

Reply:
No you can keep it. I see no harm in a sanity check.